apps/api/src/view-models/buildViewModelForResource/viewModels/playlist.view-model.ts

```diff
@@ -66,4 +66,5 @@ export class PlaylistViewModel extends BaseViewModel implements IPlayListViewMod
             )
         );
     }
+    url: string;
```

Comment:
Remove this. As I said before, the url property comes from `PlaylistEpisode`, which has a `mediaItemUrl` property.

Reply:
The call to `allResources.flatMap( resource => resource.buildEpisodes(...))` results in an episodes property on the view model that has an array of `PlaylistEpisodes`. In the playlist detail, you generate a list of these `PlaylistEpisodes`, and bind the `mediaItemUrl` property to the audio panel on click.

Reply:
this has been addressed